go-controller/pkg/ovn/ovn.go

```diff
@@ -731,6 +731,11 @@ func nodeChassisChanged(oldNode, node *kapi.Node) bool {
 	return oldChassis != newChassis
 }
 
+// disableGatewayMtuChanged returns true if the disableGatewayMtu annotation on the node was updated.
+func disableGatewayMtuChanged(oldNode, node *kapi.Node) bool {
+	return util.ParseNodeDisableGatewayMtu(oldNode) != util.ParseNodeDisableGatewayMtu(node)
```

Comment:
use consistent case for acronym: https://github.com/golang/go/wiki/CodeReviewComments#initialisms

Reply:
fixed